src/Factory.php

```diff
@@ -7,12 +7,12 @@ class Factory
     public static function create()
     {
         // @codeCoverageIgnoreStart
-        if (function_exists('event_base_new')) {
-            return new LibEventLoop();
-        } elseif (class_exists('libev\EventLoop', false)) {
+        if (class_exists('libev\EventLoop', false)) {
             return new LibEvLoop;
         } elseif (class_exists('EventBase', false)) {
             return new ExtEventLoop;
+        } elseif (function_exists('event_base_new') && version_compare(PHP_VERSION, '7.0.0', '<')) {
+            return new LibEventLoop();
```

Comment:
Makes perfect sense to lower priority here and do not pick this extension on PHP 7+ at all :+1: